.github/ISSUE_TEMPLATE/visual_design_user_story.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 🧑‍🎨 Visual Design User story
+about: (DEV TEAM ONLY) A small chunk of work to be done by Visual Designer
+title: '(Fully descriptive title)'
+labels: 'visual design,triage'
+---
+
+<!-- Ensure the title can be understood without the parent item's context, e.g. "nimble-datepicker Visual Design" rather than just "Visual Design" -->
+
+## 📌 Request Background
+
+<!-- A short description of the request and requester. E.g. Who is the client and how do they expect to use the component? -->
+<!-- Link to IxD spec -->
+
+## Acceptance Criteria
+
+- [ ] Visual Design Spec created and reviewed (link)
```

Comment:
If we decide to capture "handoff to dev" as a task for the IxD spec we should do the same for the VxD spec here.